src/index.js

```diff
@@ -0,0 +1,73 @@
+import './style.css';
+import Icon from './loading.png';
+import Icon2 from './verticaldots.png';
+
+const mainList = document.getElementById('main-list');
+const loadingIcon = new Image();
+loadingIcon.src = Icon;
+
+const taskArray = [{
+  description: 'Complete day 1 milestone',
+  completed: false,
+  index: 2,
+},
+{
+  description: 'Do the laundry',
+  completed: false,
+  index: 3,
+},
+{
+  description: 'Wash the dishes',
+  completed: false,
+  index: 1,
+},
+];
+
+function sort() {
+  taskArray.sort((a, b) => a.index - b.index);
+}
+
+function paintList() {
+  sort();
+  const titleContainer = document.createElement('div');
+  const paragraph = document.createElement('p');
+  titleContainer.classList.add('title-container');
+  paragraph.classList.add('list-tittle');
+  paragraph.innerHTML = "Today's To Do";
+  titleContainer.appendChild(paragraph);
+  titleContainer.appendChild(loadingIcon);
+  const addInput = document.createElement('input');
```

Comment:
- [ ] It would be better if we add the `icon` in the input field as we have in the required design. Here is an image description
Current Project
![Screenshot from 2021-09-22 00-41-31](https://user-images.githubusercontent.com/46388113/134265816-67c1498f-ee0a-44fc-8562-02d9b95de550.png)
Required Design
![Screenshot from 2021-09-22 00-41-59](https://user-images.githubusercontent.com/46388113/134265848-c1a1d15f-1afb-45f1-a0d7-b6f3e80033ec.png)